Update rhacs RBAC resource list (#58080)
openshift/openshift-docs: modules/rbac-resource-definitions.adoc

| Add new secured clusters and modify or delete existing clusters.

| Compliance
| View compliance standards and results, as well as recent compliance runs and associated completion status.
| Trigger compliance runs
//TODO: Add link to Compliance

| Deployment
| View deployments (workloads) in secured clusters.
| N/A

| DeploymentExtension

> | View

Reviewer: ```suggestion
a| View the following items:
```

> * process baselines.

Reviewer: ```suggestion
* Process baselines
```

> * process activity in deployments.

Reviewer: ```suggestion
* Process activity in deployments
```

> * risk results.

Reviewer: ```suggestion
* Risk results
```

> | Modify

Reviewer: ```suggestion
a| Modify the following items:
```

> * process baselines (add or remove processes).

Reviewer: ```suggestion
* Process baselines (add or remove processes)
```

//TODO: Add link to pprocess baselines
//TODO: Add link to Risks

| Detection
| Check build-time policies against images or deployment YAML.
| N/A
//TODO: Add link to Integrate with CI systems

| Image
| View images, their components, and their vulnerabilities.
| N/A
//TODO: Add link to VIew images topic

| Integration

> | View:

Reviewer: ```suggestion
a| View the following items:
```

> * existing API tokens.

Reviewer: ```suggestion
* Existing API tokens
```

> * existing integrations with automated backup systesm such as AWS S3.

Reviewer: ```suggestion
* Existing integrations with automated backup systems such as Amazon Web Services (AWS) S3
```

> * existing image registry integrations.

Reviewer: ```suggestion
* Existing image registry integrations
```

> * existing integrations for notification systems like email, Jira or webhooks.

Reviewer: ```suggestion
* Existing integrations for notification systems like email, Jira, or webhooks
```

> | Modify

Reviewer: ```suggestion
a| Modify the following items:
```

> * API tokens (create new or revoke existing ones).

Reviewer: ```suggestion
* API tokens (create new tokens or revoke existing tokens)
```

> * the configurations of backup integrations.

Reviewer: ```suggestion
* The configurations of backup integrations
```

> * image registry integrations (create/edit/delete).

Reviewer: ```suggestion
* Image registry integrations (create/edit/delete)
```

> * notification integrations (create/edit/delete).

Reviewer: ```suggestion
* Notification integrations (create/edit/delete)
```

//TODO: Add link to API tokens
//TODO: Add link to AWS S3 integration
//TODO: Add link to image registry integrations

| K8sRole

> | View roles for Kubernetes role-based access control in secured clusters.

Reviewer: ```suggestion
| View roles for Kubernetes RBAC in secured clusters.
```

> | View users and groups for Kubernetes role-based access control in secured clusters.

Reviewer: ```suggestion
| View users and groups for Kubernetes RBAC in secured clusters.
```
